django/db/models/base.py

```diff
@@ -244,13 +244,21 @@ def __new__(cls, name, bases, attrs):
                     field = None
                 new_class._meta.parents[base] = field
```

Comment:
Ideally I believe parent links defined on abstract models should never end up in `_meta.parents` in the first place but I believe the proposed changes are less invasive considering this will be backported to a stable release.